### ISSUE TITLE
Missing SUBSYSTEM in Lengthy fixing-soft-error 

### ISSUE THREAD

**Issue by mommsen** [cmsdaq/DAQExpert#222]
The shifter reported tonight in the [elog](http://cmsonline.cern.ch/cms-elog/1044111) that the DAQExpert did not report the sub-system which is in lengthy fixing-soft-error:
```
Lengthy fixing-soft-error 2018-05-23 02:25:18
Level zero in FixingSoftError longer than 30 sec. This is caused by subsystem(s) {{SUBSYSTEM}}
```

**Comment by gladky** [comment]
Here is what happened:

1. There were 2 periods where L0 was in `fixing soft error` state, 1st 14 sec, 2nd 21 sec, between them there was ~35 msec Running state.
2. Expert saw this as 1 period where L0 was in `fixing soft error` state - 35 sec
3. The threshold for firing `Lengthy fixing-soft-error` was 30 sec. This means that 2:25:18 o'clock the condition was satisfied.
4. The problem was that no system was in `fixing soft error` state at this moment (see the attached screenshot of run info timeline. More specifically ECAL was back in running but L0 still indicated `fixing soft error` This situation lasted for 6 seconds and includes transition of TCDS from `Paused` to `Running` (via `TTCHardResetting` and `Resuming`). This is a reason why DAQExpert could not fill the problematic SUBSYSTEM information in the message.



### Why L0 was in `fixing soft error` even though there was no subsystem in this state? 

If this is expected we need to update the logic of LM to include this assumption.



![screen shot 2018-05-23 at 10 38 56](https://user-images.githubusercontent.com/1798923/40413224-93567a24-5e75-11e8-8e96-c2440c3e2386.png)

**Comment by hsakulin** [comment]
Soft error recovery always proceeds as follows: Pause TCDS, send fixSoftError to Subsystem(s), resume TCDS.

Always having a subsystem in fixingSoftError state is a wrong assumption.

**Comment by gladky** [comment]
Fixed with 1bcef7a as 2.10.7
